graphql/user/fields.ts

```diff
@@ -169,6 +169,18 @@ export class UserFieldsResolver {
         return getAppointmentsForUser(user, take, skip, cursor, direction);
     }
 
```

Comment:
Shouldn't take and skip be mandatory then in appointments(...)?

Reply:
I defaulted in the getAppointmentsForUser, but you're right the API is cleaner this way.
Will try it locally in a sec and then push the update.